screen.py

```diff
@@ -1,16 +1,22 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
might be worth adding future import just to cover if default env ends up being python 3.

ie. from __future__ import print_function etc ten use python 3 print functions